src/app/utils/error-handler.ts

```diff
@@ -5,14 +5,17 @@ import { ExternalDataFetchError } from '@/services/hapi-fhir.service';
 import { SHLinkValidationError } from '@/usecases/shlinks/validate-shlink';
 
 import { AuthenticationError } from './authentication';
+import { Logger } from './logger';
 import {
   BAD_REQUEST,
   PRECONDITION_FAILED,
   SERVER_ERROR,
 } from '../constants/http-constants';
 
-export function handleApiValidationError(error: unknown) {
-  console.error('API route error:', error);
+const logger = new Logger()
+
+export function handleApiValidationError(error: unknown, route?:string) {
+  logger.log(`API route error: ${error}`, route);
```

Comment:
the second parameter takes a logger type, route would be an invalid log type. ensure that you are passing error type